cli/alert.go

```diff
@@ -14,16 +14,21 @@ import (
 	"github.com/prometheus/alertmanager/pkg/parse"
 )
 
-var (
-	alertCmd      = app.Command("alert", "View and search through current alerts")
-	alertQueryCmd = alertCmd.Command("query", "View and search through current alerts").Default()
-	expired       = alertQueryCmd.Flag("expired", "Show expired alerts as well as active").Bool()
-	silenced      = alertQueryCmd.Flag("silenced", "Show silenced alerts").Short('s').Bool()
-	alertQuery    = alertQueryCmd.Arg("matcher-groups", "Query filter").Strings()
-)
+type alertQueryCmd struct {
+	expired, silenced bool
+	matcherGroups     []string
+}
 
-func init() {
-	alertQueryCmd.Action(queryAlerts)
+func configureAlertCmd(app *kingpin.Application, longHelpText map[string]string) {
+	var (
+		a             = &alertQueryCmd{}
+		alertCmd      = app.Command("alert", "View and search through current alerts")
+		alertQueryCmd = alertCmd.Command("query", "View and search through current alerts").Default()
```

Comment:
I find it a bit confusing to have both `alertQueryCmd` as a local variable here and as a global struct.